js/widgets/forms/clearButton.js

```diff
@@ -24,55 +27,75 @@ define( [
 				this.options.clearBtn = true;
 			}
 
-			if ( !!this.options.clearBtn && this.inputNeedsWrap ) {
-				this._addClearBtn();
+			// We do nothing on startup if the options is off or if this is not a wrapped input
+			if ( !this.options.clearBtn || !this.inputNeedsWrap ) {
```

Comment:
Unless im missing something we only dont wrap in the case of textareas, if this is the case `this.inputNeedsWrap` and `this.isTextarea` do the exact same thing.